### DISCUSSION TITLE
Next steps after the first release

### DISCUSSION THREAD

**Discussion by igor-petruk** [igor-petruk/oatbar#16] [Ideas]
Next things I plan to do:

- Tray support, medium amount of work, I have to figure out the xembed and tray protocols in X11. Each time I do something like this I try to learn about common struggles on other bars and improve upon that
- Wayland support, that will be a lot of work, so not sure on the timeline
- Easy initial configurator wizard to build a config answering questions
- GUI configurator for variables that you can use in widget properties, so you can WISYWIG configure your bar in real time and save the colors, fonts, sizes into text config automatically. For example you could use color picker dialog to select block color in real-time and tune it until ti looks perfect.
- More modules like music, improvements to network module. I also have a really unique idea of how to make a memory usage module 
- Keyboard focus on blocks. Not sure what exactly would be useful here, ideas? I would like to understand user workflows

wdyt?  Is there anything else you always wanted, but other bars don't have it? Is there anything else you always wanted, but other bars don't have it?

**Comment by 0323pin** [comment]
Keep-up the good work.
This is awesome 👍
